package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jun 21 06:16:12 UTC 2016 - igonzalezsosa@suse.com
+
+- Consider AutoYaST keep_install_network as set to 'yes'
```

Comment:
The value is actually _true._ Let's use that since other config files allow 'yes' and we should not add to the confusion.